src/cultionet/scripts/config.yml

```diff
@@ -2,11 +2,7 @@ image_vis:
   - evi2
   - gcvi
   - kndvi
-
-# The regions to process (start, end)
-regions:
-  - 1
-  - 1
+  - green
```

Comment:
@nnguyen622 can you explain this one? This would impact the example data, e.g.

Reply:
@jgrss , In this example, I produced data with `evi2, gcvi, kndvi, and green indices`. I removed region part because I hard-coded the geo_id list (as seen below) 
However, I realized that the config.yml can be changed whenever users generate training files. As such, maybe we don't need to change the config.yml in this PR?

Reply:
Ahh, that's my fault. I completely missed that the 'regions' was removed. Let's see if we can leave the config 'regions' as an option and add your request from the CLI. What do you think?

Reply:
@jgrss , I should have provided more context. I think your suggested code will allow `regions` to still be in the yml file 👍 
I will update once I test out the code generating training files